torch_agents/cogment_verse_torch_agents/hf_sb3/sb3_adapter.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2021 AI Redefined Inc. <dev+cogment@ai-r.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import copy
+import logging
+from collections import namedtuple
+
+import cogment
+
+import torch
+from cogment.api.common_pb2 import TrialState
+from cogment_verse import AgentAdapter
+
+# from cogment_verse import MlflowExperimentTracker
+from cogment_verse_torch_agents.utils.tensors import tensor_from_cog_obs
+from data_pb2 import (
+    ActorParams,
+    AgentAction,
+    AgentConfig,
+    EnvironmentConfig,
+    EnvironmentParams,
+    EnvironmentSpecs,
+    TrialConfig,
+    SimpleSB3TrainingRunConfig,
+)
+from huggingface_sb3 import load_from_hub
+from stable_baselines3 import PPO
+
+# from stable_baselines3.common.evaluation import evaluate_policy
+
+SimpleSB3Model = namedtuple("SimpleSB3Model", ["model_id", "version_number", "policy_network"])
+
+log = logging.getLogger(__name__)
+
+# pylint: disable=arguments-differ
+class SimpleSB3AgentAdapter(AgentAdapter):
+    def __init__(self):
+        super().__init__()
+        self._dtype = torch.float
+
+    @staticmethod
+    async def run_async(func, *args):
+        """Run a given function asynchronously in the default thread pool"""
+        event_loop = asyncio.get_running_loop()
+        return await event_loop.run_in_executor(None, func, *args)
+
+    def _create_actor_implementations(self):
+        async def impl(actor_session):
+            actor_session.start()
+
+            config = actor_session.config
+
+            # checkpoint = load_from_hub(
```

Comment:
Can we remove this commented part now? Presumably this is a leftover from testing.